server/src/servershot.js

```diff
@@ -331,7 +331,7 @@ class Shot extends AbstractShot {
 Shot.getRawBytesForClip = function(uid) {
   return db.select(
     `SELECT images.url, images.contenttype, data.deviceid, devices.accountid
-     FROM images, devices
+     FROM devices, images
```

Comment:
I don't think it matters either way, but any reason these got switched?

Reply:
Otherwise it's `devices JOIN data ON images.shotid = data.id`